docs/clusters/blanca/blanca.md

```diff
@@ -258,7 +257,7 @@ module load python
 python myscript.py
 ```
 
-Same as Example 1, but specify desired node features, in this case the avx2 instruction set and Redhat V7 OS (job will run on any node meeting these feature requirements, and which has at least 16 cores per node).
+Same as Example 1, but specify desired node features, in this case the avx2 instruction set and RedHat V7 OS (job will run on any node meeting these feature requirements, and which has at least 16 cores per node).
```

Comment:
I don't believe the RHEL 7 constraint exists anymore. All Blanca nodes are now RHEL 8. I would look into this and adjust it appropriately.